modules/util/css-builder/src/test/java/com/liferay/css/builder/CSSBuilderTest.java

```diff
@@ -91,11 +90,18 @@ public void testCssBuilderWithFragmentChange() throws Exception {
 
 		_changeContentInPath(fragmentFileToChange, "brown", "khaki");
 
-		try (CSSBuilder cssBuilder = new CSSBuilder(
-				_docrootDirName, false, ".sass-cache/",
-				_PORTAL_COMMON_CSS_DIR_NAME, 6, new String[0], "jni")) {
+		CSSBuilderArgs args = new CSSBuilderArgs();
```

Comment:
Again, since the variable is of type `CSSBuilderArgs`, it should be called `cssBuilderArgs`.